### ISSUE TITLE
Unknown class Sentry

### ISSUE THREAD

**Issue by gemal** [shipmonk-rnd/composer-dependency-analyser#89]
I've fixed almost all of my composer problems and is now down to a few ones

it reports this:
```
Sentry\captureException
      app\src\Network\SentryGateway.php:135
```

line 135 is `\Sentry\captureException($exception);`

is the tool wrong or my code?

According to the docs the usage is correct:
https://docs.sentry.io/platforms/php/usage/

using your latest version

**Comment by janedbal** [comment]
This is a bug, it should not report functions as unknown classes. Thanks for a report

**Comment by janedbal** [comment]
Can you test `dev-master`? It should work there.

**Comment by gemal** [comment]
just tested. works perfect. thanks for the work!

**Comment by janedbal** [comment]
Fixed in [1.3.1](https://github.com/shipmonk-rnd/composer-dependency-analyser/releases/tag/1.3.1)

**Comment by gemal** [comment]
> Fixed in [1.3.1](https://github.com/shipmonk-rnd/composer-dependency-analyser/releases/tag/1.3.1)

Perfect. It fixed a few mistakes in our composer.json file
Your program is now part of our automated CI process!
Thanks for great open source work!
